src/main/java/stirling/software/SPDF/config/security/CustomAuthenticationSuccessHandler.java

```diff
@@ -18,15 +17,10 @@
 public class CustomAuthenticationSuccessHandler
         extends SavedRequestAwareAuthenticationSuccessHandler {
 
-    @Autowired private LoginAttemptService loginAttemptService;
-
     @Override
     public void onAuthenticationSuccess(
             HttpServletRequest request, HttpServletResponse response, Authentication authentication)
             throws ServletException, IOException {
-        String username = request.getParameter("username");
-        loginAttemptService.loginSucceeded(username);
```

Comment:
Is this meant to be removed?

Reply:
There is a separate routine for oauth in `CustomOAuth2AuthenticationSuccessHandler`

Reply:
Then why remove this?